src/Dfe.EarlyYearsQualification.Mock/Content/MockContentfulService.cs

```diff
@@ -376,6 +376,46 @@ await Task.FromResult(CreateDropdownPage()),
                                      });
     }
 
+    public Task<CannotFindQualificationPage?> GetCannotFindQualificationPage(int level, int startMonth, int startYear)
+    {
+        var backButton = new NavigationLink
+                         {
+                             DisplayText = "TEST",
+                             Href = "/qualifications",
```

Comment:
Suggestion (SonarCloud): there are now 4 instances of `"/qualifications"` in this class, can we pull them out into a `const string`?

Reply:
Created a const now :-)